queue/private.php

```diff
@@ -176,7 +179,7 @@ class="wdn-button wdn-button-triad ended"
 			</form>
 		</div>
 
-		<div class="client-appointmentId"><b>Appointment ID: </b>{{client.appointmentId}}</div>
+		<div class="client-appointmentId dcf-txt-xs dcf-txt-right"><b>Appointment ID: </b>{{client.appointmentId}}</div>
```

Comment:
We might consider adding just a little bottom padding on this--it was really close to the bottom footer.